packages/miniflare/src/plugins/core/index.ts

```diff
@@ -231,16 +235,20 @@ function getCustomServiceDesignator(
 	workerIndex: number,
 	kind: CustomServiceKind,
 	name: string,
-	service: z.infer<typeof ServiceDesignatorSchema>
+	service: z.infer<typeof ServiceDesignatorSchema>,
+	// override with router worker
+	hasAssetsAndIsVitest: boolean = false
```

Comment:
Why the `AndIsVitest`? I think we probably want to override even if not in vitest and has assets?

Reply:
This option only gets used if it's in vitest. I could also set it if not using vitest and has assets, but then its just set and doesn't get used, and I don't know if that's any better.

Ideally I would not have this in core plugin options at all and just check if the asset plugin options are set, but I don't think I can refer to the assets plugin options from another plugin. If I'm mistaken though that would make me very happy!